src/cmd-clean

```diff
@@ -7,15 +7,16 @@ dn=$(dirname "$0")
 
 print_help() {
     cat 1>&2 <<'EOF'
-Usage: coreos-assembler clean --help
+Usage: coreos-assembler clean --help [--all]
        coreos-assembler clean
```

Comment:
Maybe we should rewrite as

```suggestion
Usage: coreos-assembler clean [--help] [--all]
```

Reply:
I think what we really want is to write this stuff in not-shell and autogenerate help, but for now I think it's cleaner to add `--all` to the existing second line.